src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router'
+import React from 'react'
+
+const Breadcrumbs = props => {
```

Comment:
how do u feel about explicitly calling out the props used, i.e.,  `const Breadcrumbs = ({ params }) => { ...` ?

not saying we change this, just curious for your general thoughts here